### ISSUE TITLE
Audit Help Desk Documentation

### ISSUE THREAD

**Issue by michelle-dooley** [department-of-veterans-affairs/va.gov-cms#18541]
## User Story or Problem Statement
As help desk team supporting editor, I need to confirm the documentation that exists is accurate and that processes are defined. To do this, I first need to understand where there is out of date documentation. 

## Description or Additional Context 
There are multiple sources of Help Desk documentation (Confluence, GH, KB, etc) and we know some of it is out of date and no longer accurate.  We need to make sure we are working from the most up to date information with one single source of truth so we are providing the best support to our editors and so any team member can easily pick up and help support the help desk as needed.

## Steps for Implementation
For each system we need to find and confirm accuracy
- Review the documentation in Confluence  
- Review the documentation in Github  
- Review the documentation in KB
- Review canned responses in Jira
- Identify and review any other sources of Help Desk documentation

## Acceptance Criteria
- [x] Out-of-date/incorrect documentation has been identified.
- [x] A single source of truth has been identified.

**Comment by michelle-dooley** [comment]
[HD Documentation Audit worksheet](https://docs.google.com/spreadsheets/d/1pPwbS4gOQkKzJ9pcgEFO7puEgeQnHZOC4MElOij7Cw8/edit?gid=0#gid=0) for tracking

**Comment by gracekretschmer-metrostar** [comment]
To do: get clarity on this priority versus the AP QA work for Steve.

**Comment by srancour** [comment]
Raising this here as well as in the spreadsheet we are tracking in. While I was going through the [help desk overview page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/helpdesk/overview_wip.md) in github I found lots of links that are old/404 and some links are going to knowledge hub pages that are archived. While on those archived pages and trying to find the current link to the same topic, I realized that some of those archived pages are still linked to from pages in the active knowledge hub. One thing that needs to happen with the knowledge hub cleanup is checking the usage of the archived pages to make sure they aren't referenced anywhere else in the active pages. Example: [How to edit an existing event](https://prod.cms.va.gov/node/6485/usage).
